### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by divmain**
Adding @leerob and myself, removing @chibicode.

### CODE REVIEW

**Review by ijjk — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 11s | 11.4s | ⚠️ +342ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.014 | 2.009 | 0 |
| / avg req/sec | 1241.25 | 1244.58 | +3.33 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.25 | 1.233 | -0.02 |
| /error-in-render avg req/sec | 2000.02 | 2027.29 | +27.27 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 677f882d2ed8..5cb0.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.5 kB | 59.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| amp-e3e9bc99..932c.js gzip | 536 B | 536 B | ✓ |
| hooks-5023ef..3670.js gzip | 888 B | 888 B | ✓ |
| index-3a2b03..c677.js gzip | 227 B | 227 B | ✓ |
| link-177412b..043e.js gzip | 1.67 kB | 1.67 kB | ✓ |
| routerDirect..1a66.js gzip | 303 B | 303 B | ✓ |
| withRouter-9..1b7d.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.66 kB | 8.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 347 B | 347 B | ✓ |
| Overall change | 347 B | 347 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 609 B | 609 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 13.2s | 13.1s | -17ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 677f882d2ed8..5cb0.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.5 kB | 59.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| amp-e3e9bc99..932c.js gzip | 536 B | 536 B | ✓ |
| hooks-5023ef..3670.js gzip | 888 B | 888 B | ✓ |
| index-3a2b03..c677.js gzip | 227 B | 227 B | ✓ |
| link-177412b..043e.js gzip | 1.67 kB | 1.67 kB | ✓ |
| routerDirect..1a66.js gzip | 303 B | 303 B | ✓ |
| withRouter-9..1b7d.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.66 kB | 8.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 347 B | 347 B | ✓ |
| Overall change | 347 B | 347 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _error.js | 1.01 MB | 1.01 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| amp.amp.html | 10.4 kB | 10.4 kB | ✓ |
| amp.html | 1.86 kB | 1.86 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1.01 MB | 1.01 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.06 MB | 1.06 MB | ✓ |
| withRouter.js | 1.06 MB | 1.06 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 5 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 12.7s | 13.1s | ⚠️ +470ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.985 | 2.006 | ⚠️ +0.02 |
| / avg req/sec | 1259.7 | 1246.5 | ⚠️ -13.2 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.259 | 1.27 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1986.19 | 1968.04 | ⚠️ -18.15 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 597-d08f5fe5..d690.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 6.58 kB | 6.58 kB | ✓ |
| webpack-HASH.js gzip | 954 B | 954 B | ✓ |
| Overall change | 59.9 kB | 59.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-6e0a903..b885.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error-9e190..f9d6.js gzip | 3.38 kB | 3.38 kB | ✓ |
| amp-7fd79cc4..99ee.js gzip | 538 B | 538 B | ✓ |
| hooks-f7e199..11b9.js gzip | 904 B | 904 B | ✓ |
| index-3dc22c..ffbb.js gzip | 232 B | 232 B | ✓ |
| link-7cc9e62..ef27.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..323a.js gzip | 308 B | 308 B | ✓ |
| withRouter-a..2ef8.js gzip | 304 B | 304 B | ✓ |
| Overall change | 8.59 kB | 8.59 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| index.html gzip | 586 B | 586 B | ✓ |
| link.html gzip | 594 B | 594 B | ✓ |
| withRouter.html gzip | 580 B | 580 B | ✓ |
| Overall change | 1.76 kB | 1.76 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/",
         "query": {},
-        "buildId": "M8uiptO6Hf51yLN1kIzYl",
+        "buildId": "DsedsRu59EeD1B5BLQYyK",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_buildManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_ssgManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -48,7 +48,7 @@
         "props": { "pageProps": {} },
         "page": "/link",
         "query": {},
-        "buildId": "M8uiptO6Hf51yLN1kIzYl",
+        "buildId": "DsedsRu59EeD1B5BLQYyK",
         "isFallback": false,
         "gip": true
       }
@@ -82,11 +82,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_buildManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_ssgManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/withRouter",
         "query": {},
-        "buildId": "M8uiptO6Hf51yLN1kIzYl",
+        "buildId": "DsedsRu59EeD1B5BLQYyK",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_buildManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/M8uiptO6Hf51yLN1kIzYl/_ssgManifest.js"
+      src="/_next/static/DsedsRu59EeD1B5BLQYyK/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
</details>
Commit: af158add9f00d9441e67d3ae20fe103fb03f3470

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.4s | -274ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.442 | 2.279 | -0.16 |
| / avg req/sec | 1023.93 | 1096.81 | +72.88 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.795 | 1.723 | -0.07 |
| /error-in-render avg req/sec | 1393.03 | 1451.14 | +58.11 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 677f882d2ed8..5cb0.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.5 kB | 59.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| amp-e3e9bc99..932c.js gzip | 536 B | 536 B | ✓ |
| hooks-5023ef..3670.js gzip | 888 B | 888 B | ✓ |
| index-3a2b03..c677.js gzip | 227 B | 227 B | ✓ |
| link-177412b..043e.js gzip | 1.67 kB | 1.67 kB | ✓ |
| routerDirect..1a66.js gzip | 303 B | 303 B | ✓ |
| withRouter-9..1b7d.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.66 kB | 8.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 347 B | 347 B | ✓ |
| Overall change | 347 B | 347 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 609 B | 609 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 14.4s | 14.3s | -39ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 677f882d2ed8..5cb0.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.5 kB | 59.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| amp-e3e9bc99..932c.js gzip | 536 B | 536 B | ✓ |
| hooks-5023ef..3670.js gzip | 888 B | 888 B | ✓ |
| index-3a2b03..c677.js gzip | 227 B | 227 B | ✓ |
| link-177412b..043e.js gzip | 1.67 kB | 1.67 kB | ✓ |
| routerDirect..1a66.js gzip | 303 B | 303 B | ✓ |
| withRouter-9..1b7d.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.66 kB | 8.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 347 B | 347 B | ✓ |
| Overall change | 347 B | 347 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _error.js | 1.01 MB | 1.01 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| amp.amp.html | 10.4 kB | 10.4 kB | ✓ |
| amp.html | 1.86 kB | 1.86 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1.01 MB | 1.01 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.06 MB | 1.06 MB | ✓ |
| withRouter.js | 1.06 MB | 1.06 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 5 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| buildDuration | 14.3s | 15.1s | ⚠️ +840ms |
| nodeModulesSize | 69.8 MB | 69.8 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.345 | 2.355 | ⚠️ +0.01 |
| / avg req/sec | 1066.17 | 1061.43 | ⚠️ -4.74 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.67 | 1.641 | -0.03 |
| /error-in-render avg req/sec | 1497 | 1523.73 | +26.73 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| 597-d08f5fe5..d690.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 6.58 kB | 6.58 kB | ✓ |
| webpack-HASH.js gzip | 954 B | 954 B | ✓ |
| Overall change | 59.9 kB | 59.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _app-6e0a903..b885.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error-9e190..f9d6.js gzip | 3.38 kB | 3.38 kB | ✓ |
| amp-7fd79cc4..99ee.js gzip | 538 B | 538 B | ✓ |
| hooks-f7e199..11b9.js gzip | 904 B | 904 B | ✓ |
| index-3dc22c..ffbb.js gzip | 232 B | 232 B | ✓ |
| link-7cc9e62..ef27.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..323a.js gzip | 308 B | 308 B | ✓ |
| withRouter-a..2ef8.js gzip | 304 B | 304 B | ✓ |
| Overall change | 8.59 kB | 8.59 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | divmain/next.js update-codeowners | Change |
| - | - | - | - |
| index.html gzip | 586 B | 586 B | ✓ |
| link.html gzip | 594 B | 594 B | ✓ |
| withRouter.html gzip | 580 B | 580 B | ✓ |
| Overall change | 1.76 kB | 1.76 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/",
         "query": {},
-        "buildId": "rqVfjKRspC6yJMAIIeqeY",
+        "buildId": "hFbDLx9AWunJA5AfzXQxn",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_buildManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_ssgManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -48,7 +48,7 @@
         "props": { "pageProps": {} },
         "page": "/link",
         "query": {},
-        "buildId": "rqVfjKRspC6yJMAIIeqeY",
+        "buildId": "hFbDLx9AWunJA5AfzXQxn",
         "isFallback": false,
         "gip": true
       }
@@ -82,11 +82,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_buildManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_ssgManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/withRouter",
         "query": {},
-        "buildId": "rqVfjKRspC6yJMAIIeqeY",
+        "buildId": "hFbDLx9AWunJA5AfzXQxn",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_buildManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/rqVfjKRspC6yJMAIIeqeY/_ssgManifest.js"
+      src="/_next/static/hFbDLx9AWunJA5AfzXQxn/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
</details>
Commit: af158add9f00d9441e67d3ae20fe103fb03f3470

**Comment by ijjk** [comment]
## Failing test suites 
Commit: af158add9f00d9441e67d3ae20fe103fb03f3470

**test/integration/script-loader/test/index.test.js** 
- Script Loader > priority lazy

<details>
<summary>Expand output</summary>

  ● Script Loader › priority lazy

    thrown: "Exceeded timeout of 300000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      66 |   })
      67 | 
    > 68 |   it('priority lazy', async () => {
         |   ^
      69 |     let browser
      70 |     try {
      71 |       browser = await webdriver(appPort, '/page3')

      at integration/script-loader/test/index.test.js:68:3
      at Object.<anonymous> (integration/script-loader/test/index.test.js:21:1)

</details>

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*          @timneutkens @Timer @ijjk @lfades
```
Comment (GitBash-ux) [inline .github/CODEOWNERS:4]: *          @timneutkens @Timer @ijjk @lfades @divmain

Comment (GitBash-ux) [reply .github/CODEOWNERS:4]: /docs/     @timneutkens @Timer @ijjk @lfades @divmain @leerob

Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Add review comment
Cancel


/examples/ @timneutkens @Timer @ijjk @lfades @divmain @leerob

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-/docs/     @timneutkens @Timer @ijjk @lfades @chibicode
```
Comment (GitBash-ux) [inline .github/CODEOWNERS:5]: /docs/     @timneutkens @Timer @ijjk @lfades @divmain @leerob
Write

- [ ]

**Pull request merged**